api/HTMLImageElement.json

```diff
@@ -669,6 +669,56 @@
           }
         }
       },
+      "loading": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "76"
+            },
+            "chrome_android": {
+              "version_added": "76"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": "75"
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "See <a href='https://bugzil.la/1542784'>bug 1542784</a>"
```

Comment:
This refers to a bug that is resolved fixed and I don't see information about Firefox Android there right away. Can you clarify why this bug reference is needed or is it the wrong bug?

Reply:
@a2sheppy This PR is still blocked on this question, if you get a chance to take a look soon. Thank you!

Reply:
@a2sheppy ping

Reply:
Change submitted. Sorry about that.